javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+
+function filterOutStrings(values) {
+  var newValue = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof (values[i]) === typeof (2)) {
```

Comment:
Two things:
1. This will only save data that is a `number` data type, and we want everything that is not a `string`
2. You do not need to use `typeof` on a value to get the name of that data type, you can just use a string that holds the name of the data type, since that is what `typeof` will evaluate to. EX: `"string"`, `"number"`, etc.